tests/autoyast/validate_salt_formula.pm

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Make sure we are the salt formula has set message-of-the-day(motd)
+# Maintainer: ybontakis <ybonatakisz@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    #assert_screen("login-console-motd-message") if get_var("SALT_FORMULAS_PATH");
+    assert_script_run "grep \"Happy Testing Changed\" /etc/motd";
```

Comment:
Please, here use test data, as you can see, you have different text for the string to search (lower letters, upper letters) which probably will fail.

Reply:
actually i submit code that i changed to test failure. my bad.

Reply:
same as above